src/targets/chrome/docker.js

```diff
@@ -49,14 +49,10 @@ function createChromeDockerTarget({
   let dockerUrl = baseUrl;
   const storybookUrl = baseUrl;
   const dockerPath = 'docker';
-  const runArgs = [
-    'run',
-    '--rm',
-    '-d',
-    '--security-opt',
-    `seccomp=${__dirname}/docker-seccomp.json`,
-    '-P',
-  ];
+  const runArgs = ['run', '--rm', '-d', '-P'];
+  if (os.platform() === 'darwin') {
+    runArgs.push(`--security-opt=seccomp=${__dirname}/docker-seccomp.json`);
+  }
```

Comment:
Interesting, could you make a pre-release of it so I can try it on my CI?

Reply:
You can just npm install this branch instead

Reply:
I'm doing it, thanks :)

Reply:
Made a release for you, just install 0.6.2!